.github/workflows/update_chromium_revision.yml

```diff
@@ -0,0 +1,36 @@
+name: update_chromium_revision
+on:
+  # Trigger everyday at 10 UTC.
+  schedule:
+   - cron: 0 10 * * *
+  workflow_dispatch:
+jobs:
+  update:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: setup python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+        cache: 'pip'
+    - run:  pip install requests
+    - name: execute update_chromium_revision.py
+      run: python -m tools.wpt.update_chromium_revision
```

Comment:
This is unusual, at least in WPT's CI setup, why is the script invoked like this?

Reply:
It was a way to handle the relative imports in the file. I've refactored things so this is no longer needed.